### PR TITLE
Workaround for metrics configuration ansible playbook hanging forever

### PR DESCRIPTION
**Pull request by tinez**
We're experiencing situations where the ansible playbook for configuring
metrics is hanging forever. This problem has been analyzed and it's most
probably an internal issue of either ansible-core or python interpreter. Since
it only seems to be an OST problem let's add a timeout and retries for this.

### CODE REVIEW

File: ost_utils/utils.py [hunk 1]
```diff
@@ -31,33 +58,69 @@ def func_vector(target, args_sequence):
...
+
+    def _gather_results(self):
+        for q in self.queues:
+            self.results.append(q.get(block=False))
+
+    def _handle_exceptions(self, raise_exceptions):
+        exceptions = [result['exception'] for result in self.results if 'exception' in result]
+
+        if exceptions:
+            LOGGER.debug(f"{len(exceptions)} out of {len(self.targets)} threads raised exceptions")
```
Comment (acernek) [inline ost_utils/utils.py:120]: Maybe we could log the exceptions? Raising just the first one might prolong debugging etc.

Comment (tinez) [reply ost_utils/utils.py]: Sure, good idea

**Review by acernek — APPROVED** [review]

**Pull request merged**